src/main/kotlin/eu/europa/ec/eudi/sdjwt/SdJwtVerifier.kt

```diff
@@ -260,7 +283,7 @@ typealias JwtAndClaims = Pair<Jwt, JsonObject>
  * A single point for verifying SD-JWTs in both [Combined Issuance Format][verifyIssuance]
  * and [Combined Presentation Format][verifyPresentation]
  */
-interface SdJwtVerifier {
+interface SdJwtVerifier<out JWT> {
```

Comment:
I'm starting to feel that the methods of `SdJwtVerifier` are not properly defined. 

Parameters like `jwtSignatureVerifier`, `keyBindingVerifier` feel more like an implementation detail rather than a runtime parameter.

By the way, if we have to mark `JWT` with `@UnsafeVariance` in so many instances, is it not the case that it is not an `out` parameter of `SdJwtVerifier`?

Reply:
Indeed, it seems that the `SdJwtVcVerifier` could be renamed to `SdJwtVerifierOps` and represent, in general, the operations to verify an sd-jwt (in string or JWS JSON). In this case, to obtain such an instance, caller would  have to provide a jwtSignatureVerifier and a kbJwtVerifier.

Reply:
> By the way, if we have to mark `JWT` with `@UnsafeVariance` in so many instances, is it not the case that it is not an `out` parameter of `SdJwtVerifier`?

Marked `SdJwtVerifier` as invariant to JWT